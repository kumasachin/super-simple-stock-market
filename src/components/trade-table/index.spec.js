import { render, screen, cleanup } from '@testing-library/react';
import Trades from './';

describe("Trades", () => {
  afterEach(cleanup);

  it('should take a snapshot', () => {
    const { asFragment } = render(<Trades />)
    expect(asFragment(<Trades />)).toMatchSnapshot()
  });

  it('should take a snapshot', () => {
    const { asFragment } = render(<Trades trade={[{}]}/>)
    expect(asFragment(<Trades />)).toMatchSnapshot()
  });
});