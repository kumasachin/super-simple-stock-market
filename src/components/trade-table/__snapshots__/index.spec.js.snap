// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trades should take a snapshot 1`] = `
<DocumentFragment>
  <h2>
    Trades
  </h2>
  <table
    class="tableFormat"
  >
    <thead>
      <tr>
        <th>
          Date
        </th>
        <th>
          Stock Symbol
        </th>
        <th>
          Quantity
        </th>
        <th>
          Action
        </th>
        <th>
          Price
        </th>
        <th>
          Total Price
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</DocumentFragment>
`;

exports[`Trades should take a snapshot 2`] = `
<DocumentFragment>
  <h2>
    Trades
  </h2>
  <table
    class="tableFormat"
  >
    <thead>
      <tr>
        <th>
          Date
        </th>
        <th>
          Stock Symbol
        </th>
        <th>
          Quantity
        </th>
        <th>
          Action
        </th>
        <th>
          Price
        </th>
        <th>
          Total Price
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</DocumentFragment>
`;
